package.json

```diff
@@ -45,15 +48,20 @@
     "@types/react-swipeable-views": "0.0.27",
     "@types/react-tap-event-plugin": "0.0.30",
     "@types/redux": "3.6.0",
-    "browserify": "^13.0.1",
+    "browserify": "^13.3.0",
     "copyfiles": "^1.0.0",
+    "del-cli": "^1.1.0",
     "typescript": "2.3.2"
   },
   "scripts": {
     "compile": "node_modules/.bin/tsc -p ./",
     "bundle:content": "node_modules/.bin/browserify ./extension/instrument.js > ./extension/content.js",
     "bundle:visualization": "node_modules/.bin/browserify ./extension/app.js > ./extension/clarity.js",
-    "binplace:extension": "node_modules/.bin/copyfiles -f ./src/extension/*.html ./src/extension/manifest.json ./src/extension/*.png ./extension/",
-    "build": "npm run compile && npm run bundle:content && npm run binplace:extension && npm run bundle:visualization"
+    "binplace:extensionChrome": "node_modules/.bin/copyfiles -f ./src/extension/*.html ./src/extension/manifest.json ./src/extension/*.png ./extension/",
+    "binplace:extensionEdge": "node_modules/.bin/copyfiles -f extension/* extension-edge",
+    "convertEdge": "node_modules/.bin/copyfiles -f src/extension/edge/* extension-edge/",
+    "reorg": "node_modules/.bin/copyfiles -f extension/* extension-chrome/",
```

Comment:
>reorg": "node_modules/.bin/copyfiles -f extension/* extension-chrome/", [](start = 5, length = 71)

I think we should have extension/chrome along with extension/edge rather than extension/ be implicitly chrome. You'll also avoid this ad hoc copy if you order it that way